From fb6ea59478251fc67b355dae88b9592cdd528ba7 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Wed, 15 Aug 2018 11:33:40 +0200
Subject: [PATCH] Merge pull request #125 in SW_UGW/linux from
 bugfix/DRVLIB_SW-870-remove-wrongly-added-headerfile-lantiq_irq.h to master

* commit '8065a93601e1df696dca4276eb5d008ecc7540c9':
  remove unused file
---
 .../include/asm/mach-lantiq/grx500/lantiq_irq.h    | 26 ----------------------
 drivers/dma/mips_tc_sg.c                           |  1 -
 2 files changed, 27 deletions(-)

diff --git a/arch/mips/include/asm/mach-lantiq/grx500/lantiq_irq.h b/arch/mips/include/asm/mach-lantiq/grx500/lantiq_irq.h
deleted file mode 100644
index 141076325307..000000000000
--- a/arch/mips/include/asm/mach-lantiq/grx500/lantiq_irq.h
+++ /dev/null
@@ -1,26 +0,0 @@
-/*
- *  This program is free software; you can redistribute it and/or modify it
- *  under the terms of the GNU General Public License version 2 as published
- *  by the Free Software Foundation.
- *
- *  Copyright (C) 2010 John Crispin <john@phrozen.org>
- */
-
-#ifndef _LANTIQ_XWAY_IRQ_H__
-#define _LANTIQ_XWAY_IRQ_H__
-
-#define INT_NUM_IRQ0		8
-#define INT_NUM_IM0_IRL0	(INT_NUM_IRQ0 + 0)
-#define INT_NUM_IM1_IRL0	(INT_NUM_IRQ0 + 32)
-#define INT_NUM_IM2_IRL0	(INT_NUM_IRQ0 + 64)
-#define INT_NUM_IM3_IRL0	(INT_NUM_IRQ0 + 96)
-#define INT_NUM_IM4_IRL0	(INT_NUM_IRQ0 + 128)
-#define INT_NUM_IM_OFFSET	(INT_NUM_IM1_IRL0 - INT_NUM_IM0_IRL0)
-
-#define LTQ_DMA_CH0_INT		(INT_NUM_IM2_IRL0)
-
-#define MIPS_CPU_TIMER_IRQ	7
-
-#define MAX_IM			5
-
-#endif
diff --git a/drivers/dma/mips_tc_sg.c b/drivers/dma/mips_tc_sg.c
index 51e0b8e2174c..86f8d19f136e 100755
--- a/drivers/dma/mips_tc_sg.c
+++ b/drivers/dma/mips_tc_sg.c
@@ -30,7 +30,6 @@
 #include <linux/dma/lantiq_dmax.h>
 #include <lantiq.h>
 #include <lantiq_soc.h>
-#include <lantiq_irq.h>
 
 #include <net/datapath_proc_api.h>
 #include <linux/ltq_hwmcpy.h>
